### ISSUE TITLE
Object remains when switch camera

### ISSUE THREAD

**Issue by coidergrobe** [hiukim/mind-ar-js#410]
I'm using mind ar threejs with react, I'm facing an issue that when my camera caught an object but then i use the switchCamera() method, the object remains on the screen instead of disappear. Can you have me with this issue
Here is my code:
```
 const handleSwitchCamera = () => {
    if (mindArThreeRef.current) {
      mindArThreeRef.current.switchCamera()

      setIsUsingFrontCamera(!isUsingFrontCamera)
    }
  }
```

**Comment by hiukim** [comment]
Can you try v1.2.2 and see if problem still exists

**Comment by coidergrobe** [comment]
> Can you try v1.2.2 and see if problem still exists

I tried, the problem still exists but i solved this problem with 
```
mindArThreeRef.current.forEach((m) => {
        m.scene.visible = false
        m.switchCamera()
})
```
then set it back to true when anchor.onTargetFound()

**Comment by innovacion-now** [comment]
Hi! 

I guess I'm ok to folow on this trhead. I'm not using react but I'm facing issues to switch camera on Android ( later I will try on IOS ) and ThreeJS example. 

Is switchCamera() working with threeJs? I would like to test it with the threejs example ( https://hiukim.github.io/mind-ar-js-doc/more-examples/threejs-face-facemesh( )

Also start directly with the rear camera activated. 

Right now I'm trying to do it once camera/system is already started -> Right after **mindarThree.start();**

````
const start = async() => {
  await mindarThree.start();
  mindarThree.switchCamera();
  ...
````
Error:

> getUserMedia error DOMException: Could not start video source
> mindar-face-three.prod.js:25 Uncaught (in promise) undefined

I'm on Android 8.0 and Crhome 90.0.4430.66

pd: On desktop (using Crhome) seems to do something or at least no visible errors. First start camera with some zoom, and after a while restarts same camera ( I've two connected ) without zoom in. 

Cheers!

**Comment by hiukim** [comment]
yes, it should work on threejs. here is an example: https://github.com/hiukim/mind-ar-js/blob/master/examples/face-tracking/three.html
